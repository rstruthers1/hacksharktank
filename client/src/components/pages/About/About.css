.about-hackathon {
    font-family: Arial, sans-serif;
    line-height: 1.6;
    max-width: 800px;
    margin: auto;
    padding: 20px;
}

.about-hackathon h1, .about-hackathon h2 {
    color: #333;
}

.about-hackathon section {
    margin-bottom: 40px;
}

.sign-up-btn {
    background-color: #007bff;
    color: white;
    border: none;
    padding: 10px 20px;
    border-radius: 5px;
    cursor: pointer;
}

.sign-up-btn:hover {
    background-color: #0056b3;
}
