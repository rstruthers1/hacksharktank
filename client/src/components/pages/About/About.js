import React from 'react';
import './About.css'; // Import your CSS file

const About = () => {
    return (
        <div className="about-hackathon">
            <section className="intro">
                <h1>About Hack Team Hub</h1>
                <p>Welcome to the hub of innovation and creativity where coders, designers, and innovators come together to solve challenging problems and build amazing projects in a collaborative environment.</p>
            </section>

            <section className="what-to-expect">
                           </section>



            <section className="faqs">
                <h2>FAQs</h2>
                {/* FAQs content */}
                <p>Currently, this is just a coding exercise.</p>
            </section>


        </div>
    );
};

export default About;
