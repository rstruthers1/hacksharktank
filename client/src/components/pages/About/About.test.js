import React from 'react';
import { render, screen } from '@testing-library/react';
import About from './About';
import '@testing-library/jest-dom';
import renderer from "react-test-renderer";

describe('About', () => {

    it('should render correctly', () => {
        const tree = renderer.create(<About/>).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('renders About component', () => {
        render(<About />);
        expect(screen.getByText('About Hackathon Junction')).toBeInTheDocument();
        expect(screen.getByText('Welcome to the hub of innovation and creativity where coders, designers, and innovators come together to solve challenging problems and build amazing projects in a collaborative environment.')).toBeInTheDocument();
        expect(screen.getByText('FAQs')).toBeInTheDocument();
        expect(screen.getByText('Currently, this is just a coding exercise.')).toBeInTheDocument();
    });
});