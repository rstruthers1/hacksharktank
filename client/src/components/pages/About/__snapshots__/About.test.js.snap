// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should render correctly 1`] = `
<div
  className="about-hackathon"
>
  <section
    className="intro"
  >
    <h1>
      About Hack Team Hub
    </h1>
    <p>
      Welcome to the hub of innovation and creativity where coders, designers, and innovators come together to solve challenging problems and build amazing projects in a collaborative environment.
    </p>
  </section>
  <section
    className="what-to-expect"
  />
  <section
    className="faqs"
  >
    <h2>
      FAQs
    </h2>
    <p>
      Currently, this is just a coding exercise.
    </p>
  </section>
</div>
`;
